pkg/network/node/pod.go

```diff
@@ -483,7 +483,8 @@ func podIsExited(p *kcontainer.Pod) bool {
 
 // Set up all networking (host/container veth, OVS flows, IPAM, loopback, etc)
 func (m *podManager) setup(req *cniserver.PodRequest) (cnitypes.Result, *runningPod, error) {
-	defer metrics.PodOperationsLatency.WithLabelValues(metrics.PodOperationSetup).Observe(metrics.SinceInMicroseconds(time.Now()))
+	start := time.Now()
+	defer metrics.PodOperationsLatency.WithLabelValues(metrics.PodOperationSetup).Observe(metrics.SinceInMicroseconds(start))
```

Comment:
The deferred function's arguments are evaluated immediately, so I think this is still wrong since `metrics.SinceInMicroseconds(start)` will likely return ~0.

Reply:
yes @kyrtapz , thanks. now doing it with an inline function.